src/containers/Home.js

```diff
@@ -295,39 +255,61 @@ class Homepage extends React.PureComponent {
-              data={this.props[fieldNames.sections.topicsSection]}
+        <SideBar ref={this._sidebar}>
+          <Section anchorId="latest" anchorLabel="">
+            <LatestSection
+              data={this.props[fieldNames.sections.latestSection]}
             />
-          </Background>
-          <PodcastBoxSection />
-          <Background backgroundColor={moduleBackgounds.photography}>
-            <PhotographySection
-              data={this.props[fieldNames.sections.photosSection]}
-              moreURI="photography"
+          </Section>
+          <Section anchorId="editorPick" anchorLabel="編輯精選">
+            <EditorPicks
+              data={this.props[fieldNames.sections.editorPicksSection]}
             />
-          </Background>
-          <Background backgroundColor={moduleBackgounds.infographic}>
-            <InforgraphicSection
-              data={this.props[fieldNames.sections.infographicsSection]}
-              moreURI={`categories/${categoryConst.pathSegments.infographic}`}
+          </Section>
+          {latestTopicJSX}
+          <Section anchorId="donation-box" anchorLabel="">
+            <DonationBoxSection />
+          </Section>
+          <Section anchorId="review" anchorLabel="評論">
+            <ReviewsSection
+              data={this.props[fieldNames.sections.reviewsSection]}
+              moreURI={`categories/${categoryConst.pathSegments.reviews}`}
             />
-          </Background>
+          </Section>
+          <Section anchorId="news-letter" anchorLabel="">
+            <NewsLetterSection />
+          </Section>
+          <Section anchorId="categories" anchorLabel="議題">
+            <Background backgroundColor={moduleBackgounds.category}>
+              <CategorySection data={this.props.categories} />
+            </Background>
+          </Section>
+          <Section anchorId="topic" anchorLabel="專題">
+            <Background backgroundColor={moduleBackgounds.topic}>
+              <TopicsSection
+                data={this.props[fieldNames.sections.topicsSection]}
+              />
+            </Background>
+          </Section>
+          <Section anchorId="podcast" anchorLabel="">
+            <PodcastBoxSection />
+          </Section>
+          <Section anchorId="photography" anchorLabel="攝影">
+            <Background backgroundColor={moduleBackgounds.photography}>
+              <PhotographySection
+                data={this.props[fieldNames.sections.photosSection]}
+                moreURI="photography"
+              />
+            </Background>
+          </Section>
+          <Section anchorId="infographic" anchorLabel="多媒體">
+            <Background backgroundColor={moduleBackgounds.infographic}>
+              <InforgraphicSection
+                data={this.props[fieldNames.sections.infographicsSection]}
+                moreURI={`categories/${categoryConst.pathSegments.infographic}`}
+              />
+            </Background>
+          </Section>
```

Comment:
Q: what if we use the inner section directly without `AnchorWrapper`? Will we have any warning here?

Reply:
If we use inner section directly, the sections should be configured as :

```<LatestSection 
        data={this.props[fieldNames.sections.latestSection]}
        anchor-id="latest"
        anchor-label="最新消息"
      />
```

I think the way using `AnchorWrapper` would be easier to understand and more declarative.